.github/workflows/release.yml

```diff
@@ -0,0 +1,39 @@
+name: Publish Release Artifcats
+
+on:
+  release:
+    types: [published]
+
+permissions:
+  contents: write
+  
+jobs:
+  release:
+    name: Publish Artifcats
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          submodules: true
+          ref: ${{ github.event.release.tag }}
+
+      - uses: actions/setup-go@v5
+        with:
+          go-version: '1.20'
+
+      - name: Install libolm
+        run: sudo apt update && sudo apt-get install -y libolm-dev
+      
+      - name: Build
+        run: ./scripts/build.sh
+
+      - name: upload release assets
+        run: |
+          tar -zcvf camino-messenger-bot-linux-amd64-${{ github.event.release.tag }}.tar.gz ./build/bot 
```

Comment:
We should create more sensible filename and directory structure. Currently it creates a tarball with binary named `bot` inside a directory named `build`. I think it's better to have a binary named `camino-messenger-bot` inside the root dir (no subfolders in the tarball) or a single subfolder named `camino-messenger-bot-<version>`.